packages/tests/src/slashid.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  PersonsService,
+  OpenAPI,
+  PersonHandleType,
+  PersonCredentialsService,
+} from "../slashid";
+import { hash } from "bcryptjs";
+
+const oid = process.env.E2E_SID_ORG_ID!;
```

Comment:
This is unsafe as the whole test suite depends on these variables to be set. That is solved by creating a config object that has runtime validation built in, so it throws an error when these variables are not set. We have such an object in `playwright.config.ts`, can you please extract it and use the config here instead of accessing the variables directly?